modules/gptPreAuction.js

```diff
@@ -25,15 +26,16 @@ export const appendGptSlots = adUnits => {
       : isAdUnitCodeMatchingSlot(slot));
 
     if (matchingAdUnitCode) {
-      const adUnit = adUnitMap[matchingAdUnitCode];
-      adUnit.ortb2Imp = adUnit.ortb2Imp || {};
-      adUnit.ortb2Imp.ext = adUnit.ortb2Imp.ext || {};
-      adUnit.ortb2Imp.ext.data = adUnit.ortb2Imp.ext.data || {};
-
-      const context = adUnit.ortb2Imp.ext.data;
-      context.adserver = context.adserver || {};
-      context.adserver.name = 'gam';
-      context.adserver.adslot = sanitizeSlotPath(slot.getAdUnitPath());
+      adUnitMap[matchingAdUnitCode].forEach((adUnit) => {
+        adUnit.ortb2Imp = adUnit.ortb2Imp || {};
+        adUnit.ortb2Imp.ext = adUnit.ortb2Imp.ext || {};
+        adUnit.ortb2Imp.ext.data = adUnit.ortb2Imp.ext.data || {};
+
+        const context = adUnit.ortb2Imp.ext.data;
+        context.adserver = context.adserver || {};
+        context.adserver.name = 'gam';
+        context.adserver.adslot = sanitizeSlotPath(slot.getAdUnitPath());
+      });
```

Comment:
```suggestion
    if (matchingAdUnitCode) {
      const adserver = {
        name: 'gam',
        adslot: sanitizeSlotPath(slot.getAdUnitPath())
      };
      adUnitMap[matchingAdUnitCode].forEach((adUnit) => {
        deepSetValue(adUnit, 'ortb2Imp.ext.data.adserver', adserver);
      });
    }
```

Reply:
@dgirardi 
Thanks for review and suggestion.

I committed your GH suggestion.